File: Remove_consecutive_characters/code.cpp
class Solution{
    public:
    string removeConsecutiveCharacter(string S)
    {
        // code here.
        string str;
        char temp;
        int i=0;
        int x=1;
        while(i<S.size()){
            x=1;
            if(S[i] == S[i+1]){
                while(S[i] == S[i+x]){
                    x++;
                }
                temp = S[i];
                str.append(1, temp);
                i+=x;
            }
            else{
                temp = S[i];
                str.append(1, temp);
                i+=x;
            }
        }
        return str;
    }
};

File: Remove_consecutive_characters/code.java
//User function Template for Java

class Solution{
    public String removeConsecutiveCharacter(String S){
        
        String nw="";
        int nxt=0;
        for(int i=0;i<S.length();i++){
            char c= S.charAt(i);
            int check = (int)c;
            if(nxt != check){
             nw=nw+c;
            nxt=check;
            }
            else
            continue;
        }
        return nw;
    }
}
